hack/e2e.go

```diff
@@ -277,8 +277,8 @@ func Test() bool {
 // call the returned anonymous function to stop.
 func runBashUntil(stepName string, cmd *exec.Cmd) func() {
 	log.Printf("Running in background: %v", stepName)
-	stdout, stderr := bytes.NewBuffer(nil), bytes.NewBuffer(nil)
-	cmd.Stdout, cmd.Stderr = stdout, stderr
+	output := bytes.NewBuffer(nil)
```

Comment:
I'm wondering if we could simply use `cmd.CombinedOutput()` here? Maybe in a goroutine? It's not like we're streaming the output to stdout live, we're just capturing it to include in the final output... Not 100% sure if it will work with a command that gets interrupted but I think it just might... What do you think?